docs/cloud/managed-gc.md

```diff
@@ -12,16 +12,19 @@ lakeFS Cloud
 {: .label .label-green }
 
 {: .note}
-> Managed Garbage Collection is only available for [lakeFS Cloud](../cloud/). If you are using self-managed lakeFS, [Garbage collection](../howto/garbage-collection-index.md) is available to run manually.
+> Managed GC is only available for [lakeFS Cloud](../cloud/). 
+If you are using the self-managed lakeFS, garbage collection is [available to run manually](../howto/garbage-collection-index.md).
 
-## The benefits of using managed GC on lakeFS cloud are:
-* Advanced Engine to detect and delete objects quickly and safely
-* Managed Solution
-* Garbage Collection SLA
-* Support
+## Benefits of using managed GC
+* Expired objects are deleted quickly and safely.
+* No operational overhead
+* SLA for when your objects are deleted
+* Get support from the Treeverse team.
```

Comment:
either finish lines with dot or not (I personally prefer without dots for bullet)

Reply:
Done (I used a period only for proper sentences, so now I changed them to be non-sentence fragments)